modules/opensearch/opensearch.install

```diff
@@ -49,3 +49,10 @@ function opensearch_update_7001() {
   // Rebuild registry to load it based on opensearch.info file update.
   registry_rebuild();
 }
+
+/**
+ * Ensure libraries use the latest version of opensearch (5.0).
+ */
+function opensearch_update_7002() {
+  variable_set('opensearch_url', 'https://opensearch.addi.dk/b3.5_5.0/');
```

Comment:
Ok, not much we can do then? :)

Reply:
I have updated the case with this new information.

Do they also manage to moreinfo version in local settings.php? Because we have upgraded that several times with this method. See:

Reply:
They should manage all extern endpoints (but it might depend on local contracts with DBC). When ding2 was put in production the agreement was that they should handle it in the settings file to limit bugs in relation to admin's changing these setting before the endpoints was production ready and core supported a given version.

Reply:
The PR still updates some references to the old endpoint and makes the newest version default in the installation process (like it was done in the moreinfo updates i mentioned above).

I have asked on redmine if we should still update the variable in the update function anyway.
@cableman What do you think about this?

Also, I discovered that the trailing slash is needed. If it's missing, all searches will fail. Isn't that a bit fragile and perhaps it should be documented/changed?